staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -87,9 +87,9 @@ func RoundTripProtobufTestForScheme(t *testing.T, scheme *runtime.Scheme, fuzzin
 
 var FuzzIters = flag.Int("fuzz-iters", 20, "How many fuzzing iterations to do.")
 
-// globalNonRoundTrippableTypes are kinds that are effectively reserved across all GroupVersions
+// GlobalNonRoundTrippableTypes are kinds that are effectively reserved across all GroupVersions
 // They don't roundtrip
-var globalNonRoundTrippableTypes = sets.NewString(
+var GlobalNonRoundTrippableTypes = sets.NewString(
```

Comment:
do you have an example of where this is needed externally?

Reply:
In my own round trip testing (end-user code outside of k8s.io/*) I realize I was maintaining this list before noticing this set. 

I'm exporting this for posterity - if new non-roundtrippable types are added it shouldn't break existing code.

Reply:
Here's a concrete example:

https://github.com/knative/serving/pull/6542/files?file-filters%5B%5D=#diff-76086b4e4516fa976fde1c08e6a52a6c

Reply:
I would not want to export this in a way that any other package could modify the contents of this

Reply:
Good point I've made some changes - if you prefer something simpler (ie. copying the set) let me know